### PR TITLE
Feature: Form-R part-a

### PR DESCRIPTION
**Pull request by babul3012**

### CODE REVIEW

**Comment by hihilary** [comment]
There're some conflicts with master due to some other merges. Please merge from master first.

File: src/__snapshots__/App.snapshot.spec.tsx.snap [hunk 1]
```diff
@@ -10,7 +10,7 @@ Array [
       className="nhsuk-width-container nhsuk-header__container"
     >
       <div
-        className="nhsuk-header__logo"
+        className="nhsuk-header__logo nhsuk-header__logo--only"
```
Comment (hihilary) [inline src/__snapshots__/App.snapshot.spec.tsx.snap:13]: May I ask what this snapshot file used for, please?

Comment (babul3012) [reply src/__snapshots__/App.snapshot.spec.tsx.snap:13]: This is part of snapshot testing - https://jestjs.io/docs/en/snapshot-testing
These snapshots help us to make sure that UI is not changed until we wanted it to.

**Comment by babul3012** [comment]
> There're some conflicts with master due to some other merges. Please merge from master first.

Done.

File: src/services/TraineeProfileService.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { AxiosResponse } from "axios";
+import ApiService from "./apiService";
+import { TraineeProfile } from "../models/TraineeProfile";
+
+export class TraineeProfileService {
+  private apiService: ApiService;
+  constructor() {
+    super("/trainee/api");
```
Comment (hihilary) [inline src/services/TraineeProfileService.ts:8]: This TraineeProfileService doesn't have a parent class, but calls super() in constructor. And SonarCloud reports a bug here as well.

Comment (babul3012) [reply src/services/TraineeProfileService.ts]: Updated.

**Pull request merged**